test/e2e/test_group.py

```diff
@@ -18,7 +19,7 @@ def setUp(self) -> None:
         Creates a connection to DSP-API.
         For each test method, a new TestCase instance is created, so setUp() is executed before each test method.
         """
-        self.con = Connection(server="http://0.0.0.0:3333")
+        self.con: Connection = ConnectionLive(server="http://0.0.0.0:3333")
```

Comment:
just out of curiosity: why do you type that as Connection? In the other tests you don't.

Reply:
just an experiment I forgot to remove :)